ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1398,6 +1401,37 @@ def test_ipahealthcheck_trust_agent_member_check(self):
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == self.master.hostname
 
+    def test_ipahealthcheck_with_external_ad_trust(self):
+        """
+        This testcase checks that when external trust is configured
+        between IPA and AD tree domain, IPATrustDomainsCheck
+        doesnot display ERROR
+        """
+        error_msg = (
+            "{sssctl} {key} reports mismatch: sssd domains {sssd_domains} "
+            "trust domains {trust_domains}"
+        )
+        tasks.configure_dns_for_trust(self.master, self.tree_ad)
+        tasks.establish_trust_with_ad(
+            self.master, self.ad_treedomain,
+            extra_args=['--range-type', 'ipa-ad-trust', '--external=True'])
+        self.master.run_command(
+            ['ipa', 'trust-find', '--all', '--raw']
+        )
+        returncode, data = run_healthcheck(
+            self.master,
+            "ipahealthcheck.ipa.trust",
+            "IPATrustDomainsCheck",
+            "--debug"
```

Comment:
The fourth argument of run_healthcheck is output_type (defaulting to json).

I was suggesting making a temporary change directly to run_healthcheck, maybe like:

cmd = ["ipa-healthcheck", "--debug"]

and temporary. It may produce copious output that we'd rarely need but I guess we'll see.

Reply:
Added the --debug option to the run_healthcheck function